### DISCUSSION TITLE
check_model(model) outputs different graph than README.md(?)

### DISCUSSION THREAD

**Discussion by rempsyc** [easystats/performance#336] [Q&A] [answered]
I made a tutorial introducing `check_model(model)` and at some point realized that it was giving me a different figure than that shown in the [README.md](https://github.com/easystats/performance), like this:

```
# load packages
library(performance)
library(see)

# defining a model
model <- lm(mpg ~ wt + am + gear + vs * cyl, data = mtcars)

# checking model assumptions
check_model(model)
```

From the README.md:

![image1](https://raw.githubusercontent.com/easystats/performance/master/man/figures/unnamed-chunk-15-1.png)

From `performance_0.7.2.1`:

![image2](https://i.stack.imgur.com/HNQdf.png)

I thought it was because the README.md reflected the development version from Github and not the CRAN version so I tried `install_github("easystats/performance")` and now `sessionInfo()` gives me 

```
other attached packages:
[1] performance_0.7.2.1 see_0.6.1  
```
Are these the most up-to-date?

The differences are really minor but I was just hoping them to look exactly the same especially since the one in the README.md looks better. For example, (1) One has a caption for multicollinearity (although it is cut); (2) It also has confidence bands around the linearity and variance assumptions; (3) The order of presentation is different (normality at the end and on the same row, making it easier to quickly compare, same for linearity/variance); (4) Influential observations seem to make more sense to guide interpretation (including caption).; (5) The multicollinearity is horizontal rather than vertical, therefore saving space (as well as giving reference values); (6) The interpretation guidelines differ between the two: e.g., for linearity and variance, one emphasizes the form of the line, the other the dots in relation to the line.

**Comment by bwiernik** [comment]
From the README has the most recent plots for lm models. Your example shows older versions, namely for the influential observations/outliers plot, which was previously a bar plot and is now a leverage-residuals plot, which is more useful for determining if any point is overly influential. You should update to the most recent version of performance and re-run the function.

**Reply by rempsyc** [reply]
Last time I simply used `install_github("easystats/performance")` to update but this time I completely uninstalled `performance` and reinstalled it using the same command. Same problem. Also same version:

```
other attached packages:
[1] performance_0.7.2.1
```

Could you tell me which version is actually the latest?

UPDATE: I tried also updating package `see` (to version `see_0.6.4`), and that fixed the issue! So turns out updating performance wasn't enough, one also had to update `see`! Thanks!

**Comment by rempsyc** [comment, marked as answer]
Solution: Also update package `see` to the latest version!

**Comment by bwiernik** [comment]
Be sure that you also have the current version of the `see` package installed.

To install all of the latest versions of the easystats packages use `easystats::easystats_install_latest()`
